docs/changelog.rst

```diff
@@ -8,6 +8,69 @@ This section lists features and improvements of note in each release.
 
 The full release history can be viewed `at the GitHub perses releases page <https://github.com/choderalab/perses/releases>`_.
 
+0.10.2 - Release
+----------------
+
+Bugfix release.
+
+Enhancements
+^^^^^^^^^^^^
+
+* Added upport for reading input files (ex yaml, sdf, pdbs) from AWS, GCP, and Azure. See the documentation for [cloudpathlib](https://cloudpathlib.drivendata.org/stable/authentication/) for how to setup authentication. Currently only reading the yaml from S3 is unit tested (ie `perses-cli --yaml s3://perses-testing/template_s3.yaml`), but other cloud providers and input files should work. Please report any issues on our issue tracker!  by @mikemhenry in https://github.com/choderalab/perses/pull/1073
```

Comment:
"upport" -> "support"

Do we document how you specify AWS, GCP, and Azure files in the CLI or docs or somewhere?
Also, do _all_ input files work this way, including those paths specified in the YAML? Or just `--yaml`?

Reply:
All input files, I mention `--yaml` since that is the only one we test. I will add a note to see cloud documentation on the format of the URI.